<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="Settings">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>361</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QCheckBox" name="txtCheck">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>.txt</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="xmlCheck">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.xml</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="xamlCheck">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.xaml</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="xmsCheck">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>xms</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="jsCheck">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.js</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="xslCheck">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.xsl</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cssCheck">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.css</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="phpCheck">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.php</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="jsonCheck">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.json</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="pyCheck">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.py</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="iniCheck">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.ini</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="logCheck">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.log</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="htaCheck">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.hta</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="batCheck">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.bat</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cCheck">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>150</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.c</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cppCheck">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.cpp</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="selectedExtends">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>301</width>
       <height>91</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="usersExtend">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addExtend">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>250</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбрать все</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reset">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Сбросить все</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startSearch">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Начать поиск</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="xmdCheck">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.xmd</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="htmlCheck">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.html</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="searchDirectory">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseDirectory">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="output">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>511</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Директория поиска</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subString">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Что искать</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
